### PR TITLE
Feature/1.1.4.2/create login page

### PR DESCRIPTION
**Pull request by tristan-88**
## Accomplishment:
 - Set up the login page according to the wireframe created by Kate our UX/UI Designer
 - Change the favicon to the Seattle Humane logo
 - removed unnecessary components/elements
 - duplicated the auth Google button at least for basic functionality on redirecting to designated auth pages.
 - change the initial test to look for "Pet Assistance and Welfare System" vs "Hello Isabel"
 - Added a couple of tests to test rendering buttons 
 ### Wireframe
 https://www.figma.com/file/BboJuNpVaUcHlA8ELX5MOu/Seattle-Humane-DAS?type=design&node-id=0-1&mode=design&t=BUUBxuqAZLFEEk2h-0
 ## Log-in Page
 
Screenshot 2024-02-21 at 4 48 28 AM

### Google 
Screenshot 2024-02-20 at 3 46 47 AM

### Microsoft
Screenshot 2024-02-20 at 3 46 28 AM

## What still needs to be done?
- Partner up with Yulia for test coverage for the buttons' functionality and redirect to "/"

### CODE REVIEW

File: __tests__/login.test.js [hunk 1]
```diff
@@ -38,6 +38,23 @@ describe('LoginPage', () => {
...
+          <LoginPage />
+        </UserContext.Provider>
+      </LayoutProvider>
+    );
+    
+    expect(screen.getByLabelText('Sign in with Microsoft')).toBeInTheDocument();
+  });
+
+});
+
```
Comment (samhhenderson) [inline __tests__/login.test.js:60]: Let's put this test in a __test__/pages/login folder (so we can find it when we have lots of tests)

File: pages/auth/login/index.tsx [hunk 2]
```diff
@@ -1,98 +1,108 @@
...
+  useEffect(() => {
+    supabaseClient.auth.onAuthStateChange(async (event, session) => {
+      supabaseClient.auth.getUser().then((userResponse) => {
+        const loggedInUser = userResponse.data.user;
+        if (loggedInUser) {
+          router.push("/");
+        }
+      });
+    });
+  }, [router]);
```
Comment (samhhenderson) [inline pages/auth/login/index.tsx:33]: This is ok for now. On next PR let's abstract the supabaseClient stuff into authService.

File: pages/auth/login/index.tsx [hunk 3]
```diff
@@ -1,98 +1,108 @@
...
+                    className="mr-2"
+                  />
+                }
+                label="Sign in with Microsoft"
+                className="w-full text-sm bg-white border-gray-900 text-gray-900 ml-2 pl-3 pr-5"
+                onClick={() => {
+                  authService.signInWithAzure();
+                }}
+                style={{ whiteSpace: "nowrap" }}
+              ></Button>
```
Comment (samhhenderson) [inline pages/auth/login/index.tsx:90]: There is a tricky hydration bug happening with Button when you use the browser 'back' button (but NOT the google/microsoft 'back') to get back here. I don't know enough about hydration to know what to do about it yet, but put this as another problem to tackle on the next PR. Problem goes away when you comment away the Buttons.

**Review by samhhenderson — CHANGES_REQUESTED** [review]
1. Please move the test.
2. A couple issues that we should work on next round (but let's move forward on it for now).

File: pages/auth/login/index.tsx (hunk 2 above)
Comment (tristan-88) [reply pages/auth/login/index.tsx:33]: Will do

File: pages/auth/login/index.tsx (hunk 3 above)
Comment (tristan-88) [reply pages/auth/login/index.tsx:90]: Hmm, might need to go over this. Is it expected that when we press back button from either microsoft or google suppose to go back to our log-in page?

**Review by samhhenderson — APPROVED** [review]

**Pull request merged**